### PR TITLE
[WIP] Feature/password state

### PR DESCRIPTION
**Pull request by corrideat**

### CODE REVIEW

File: shared/domains/chelonia/chelonia.js [hunk 1]
```diff
@@ -39,13 +44,38 @@ export const ACTION_REGEX: RegExp = /^((([\w.]+)\/([^/]+))(?:\/(?:([^/]+)\/)?)?)
...
+      } */
+      encryptFn: (message) => JSON.stringify(message), /* (message, state) => {        const key = state._vm?.authorizedKeys?.find((k) => k.type === 'curve25519xsalsa20poly1305' && Array.isArray(k.perm) && k.perm.includes(GIMessage.OP_ACTION_ENCRYPTED))
+        if (!key) {
+          return JSON.stringify(message)
+        }
+        return {
+          keyId: key.id,
+          content: sbp('gi.crypto/encrypt', sbp('gi.crypto/key/deserialize', key.data), JSON.stringify(message))
+        }
+      } */
```
Comment (taoeffect) [inline shared/domains/chelonia/chelonia.js:78]: The defaults should be left as they were - simple.

Any overrides should be done outside of this file by calling `/configure`, preferably from `main.js`

**Comment by taoeffect** [comment]
Should this PR be closed in favor or #1257?

**Comment by taoeffect** [comment]
> Should this PR be closed in favor or #1257?

I think, based on the call this morning, the answer is "yes".

**Comment by corrideat** [comment]
Indeed. This is no longer needed.

**Pull request closed without merging**